### PR TITLE
Travis - consider 'created' as a running state

### PR DESCRIPTION
**Pull request by gardleopard**
The TravisBuildMonitor only considers started as running as it is now. This PR makes igor aware of the `created` state and translates that into a running build as well.

This is an edge case that I have not seen in travis, but I'm sure we will see it some day.

### CODE REVIEW

File: igor-web/src/test/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverterTest.groovy [hunk 1]
```diff
@@ -36,4 +36,22 @@ class TravisResultConverterTest extends Specification {
...
+        def state = travisBuildState
+
+        when:
+        boolean result = TravisResultConverter.running(state)
+
+        then:
+        result == expectedResult
+
+        where:
+        travisBuildState | expectedResult
```
Comment (ajordens) [inline igor-web/src/test/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverterTest.groovy]: Minor aesthetics but it's nice to line up the columns (IntelliJ reformat will do the alignment).

Also consistent to use `||` between inputs and expected results.

```
where:
travisBuildStage || expectedResult
"started"        || true
"created"        || true
```

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverter.groovy [hunk 2]
```diff
@@ -48,6 +48,19 @@ class TravisResultConverter {
                 throw new IllegalArgumentException("state: ${state} is not known to TravisResultConverter.")
                 break
         }
+    }
 
+    static Boolean running(String state) {
+        switch (state) {
+            case "created":
```
Comment (ajordens) [inline igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverter.groovy]: Will `state` always be lower case for Travis?

File: igor-web/src/test/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverterTest.groovy [hunk 3]
```diff
@@ -36,4 +36,22 @@ class TravisResultConverterTest extends Specification {
         "passed"     | Result.SUCCESS
         "errored"    | Result.FAILURE
     }
+
+    def "check if build is running"() {
```
Comment (ajordens) [inline igor-web/src/test/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverterTest.groovy]: Should have an @Unroll

File: igor-web/src/test/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverterTest.groovy [hunk 4]
```diff
@@ -36,4 +36,22 @@ class TravisResultConverterTest extends Specification {
...
+        setup:
+        def state = travisBuildState
+
+        when:
+        boolean result = TravisResultConverter.running(state)
```
Comment (ajordens) [inline igor-web/src/test/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverterTest.groovy]: Rather than setup/when/then

Could be:

```
expect:
TravisResultConverter.running(travisBuildState) == expectedResult
```

**Comment by ajordens** [comment]
LGTM outside the minor feedback above

File: igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverter.groovy (hunk 2 above)
Comment (gardleopard) [inline igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverter.groovy]: yes, as far as I know. I have never seen [this] (https://github.com/spinnaker/igor/blob/master/igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverter.groovy#L26) code fail.

File: igor-web/src/test/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverterTest.groovy (hunk 4 above)
Comment (gardleopard) [inline igor-web/src/test/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverterTest.groovy]: Awesome!

**Comment by gardleopard** [comment]
Squashed and ready:)
